tests/common/bats/harbor.bash

```diff
@@ -21,6 +24,26 @@
 # - harbor.create_pull_secret [cluster] [namespace]                             - create a pull secret for the robot set in the environment
 # - harbor.delete_pull_secret [cluster] [namespace]                             - delete a pull secret
 
+harbor._curl() {
```

Comment:
Non blocking:
If you want this to be a "do not use outside of this file" I think we should standardise on `_harbor.curl` instead.